kokoro/macos_external/kokoro_build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Is this the build for the Firebase pod? We should name this file after which pods it builds. There are going to be lots of them.